gym/dev_wrappers/lambda_reward.py

```diff
@@ -0,0 +1,95 @@
+"""Lambda reward wrappers that uses jumpy for compatibility with jax (i.e. brax) and numpy environments."""
+from typing import Callable, Optional, Union
+
+import jax.numpy as jnp
+import jumpy as jp
+import numpy as np
+
+import gym
+from gym.dev_wrappers import FuncArgType
+from gym.error import InvalidBound
+
+
+class lambda_reward_v0(gym.RewardWrapper):
```

Comment:
Please no snake_case class names. I'd even take LambdaReward_v0, or LambdaRewardV0 over this

Reply:
Afaik it was a naming convention for the new dev_wrappers (also here: https://github.com/openai/gym/pull/2853). If we want to switch to standard camel case no problem for me

Reply:
Sorry I forgot to mention that I think we are keeping the normal python style with the addition of `V0` at the end.
@RedTachyon What do you think is better:`LambdaReward_v0` or `LambdaRewardV0`